### DISCUSSION TITLE
Add a Quit button to Preferences

### DISCUSSION THREAD

**Discussion by rakleed** [rxhanson/Rectangle#1024] [Ideas]
Could you please add a Quit button to Preferences? Since if I hide the menu bar icon, then even when I open the Preferences, the app menu does not appear. And I can either close Rectangle with the hot keys `Cmd+Q` (which not all users know about), or through the Activity Monitor (which is quite inconvenient). And if there was a Quit button in Preferences, it would make it easier to close Rectangle.

**Comment by Aaron-Rumpler** [comment]
The other option would be to show the menu bar for the preferences window. The advantage there would be that it would also expose About, Check for Updates, and View Logging.

**Comment by rxhanson** [comment]
Thanks for the feedback. For what reason do you open and close the app? 

Most users will just ignore certain apps - which you could configure by using the menu bar menu, then hide the menu bar icon after. 

I don't have plans to add this in myself, but I am fine merging a good pull request with it as long as it looks good and includes placeholders in the localization files.

**Reply by rakleed** [reply]
I needed to close an app when I was testing another similar app. But I agree that this is an infrequent scenario.

On the other hand, from a UX point of view, the user should have a clear option to close the app.
